templates/base.html.twig

```diff
@@ -19,6 +19,7 @@
         {% block body %}{% endblock %}
         {% block javascripts %}
             <script src="{{ asset('build/js/app.js') }}"></script>
+			<script src="{{ asset('build/js/tweets.js') }}"></script>
```

Comment:
Bitte in app.js importieren anstatt im HTML einzubinden, damit das ganze durch die Build Pipeline laufen kann.